test/check/components.go

```diff
@@ -13,10 +13,12 @@ type Component struct {
 
 var (
 	KubeMacPoolComponent = Component{
-		ComponentName:      "KubeMacPool",
-		ClusterRole:        "kubemacpool-manager-role",
-		ClusterRoleBinding: "kubemacpool-manager-rolebinding",
-		Deployments:        []string{"kubemacpool-mac-controller-manager"},
+		ComponentName:                "KubeMacPool",
+		ClusterRole:                  "kubemacpool-manager-role",
+		ClusterRoleBinding:           "kubemacpool-manager-rolebinding",
+		Deployments:                  []string{"kubemacpool-mac-controller-manager"},
+		Secret:                       "kubemacpool-service",
+		MutatingWebhookConfiguration: "kubemacpool-mutator",
```

Comment:
ditto about this commit

Reply:
We need to add these to the component so when the function CheckComponentsRemoval is called, it will take care of Secret and MutatingWebhookConfiguration objects

Reply:
I agree, but it's not directly related to the bump.
I think it should be in separate PR, like #393 was.

Reply:
It is directly caused by the bump because the changes that has been done in the new release this is why I did not separate it to another PR.
And the e2e-workflow lane will not pass w/o those.